priv/gettext/en/LC_MESSAGES/default.po

```diff
@@ -37,7 +37,7 @@ msgstr "Welcome to Elixir School Slack!"
 #, elixir-format
 #: lib/admissions_web/templates/registrar/ineligible.html.eex:8
 msgid "ineligible_content"
-msgstr "At this time Slack invitations are reserved for contributors.\nIf you feel this is a mistake, let us know on Twitter @elixirschool."
+msgstr "At this time Slack invitations are reserved for contributors.\nIf you feel this is a mistake, let us know on Twitter @elixirschool.\nPlease note that if you have contributed for the first time to Elixir School, it might take around 30 minutes for GitHub to update their records.\nWe do apologise any inconvenience."
```

Comment:
It's better safe than sorry, let's just assume it could take an hour.  How about something like:

"it may take up to an hour for GitHub to update their records"

Reply:
Updated message in 68bd0ab3ad7473e05f8c0f75b6566bcb347d56b4